src/app/components/info/info.component.scss

```
@@ -2,3 +2,9 @@
   display: block;
   padding: 1rem;
 }
+.info-link {
+  color: darkblue; 
```

Statische Farben sollten vermieden werden.
Außerdem funktioniert das Styling nicht für bereits besuchte Links.
```suggestion
  @import "../../theme";
  $accent: map-get($app-theme, accent);
.info-link:link,
.info-link:visited {
  color: mat-color($accent);
}
```

---

Alrighto, über Theming können wir ja nochmal verhandeln

---

(und das theme liegt noch eine Ebene höher)